packages/default/scss/common/_forms.scss

```diff
@@ -219,14 +219,15 @@ $forms-invalid-color: $error !default;
             }
         }
     }
-    .k-multiselect-wrap:hover,
```

Comment:
this selector is needed for angular / react.

The correct way would be to have a k-readonly class